dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Plc::UserCourseEnrollmentsController < ApplicationController
+  #TODO: Why do I need to have the except here?
+  load_and_authorize_resource except: [:create]
+  before_action :set_plc_user_course_enrollment, only: [:show, :edit, :update, :destroy]
+
+  # GET /plc/user_course_enrollments
+  # GET /plc/user_course_enrollments.json
+  def index
+    @plc_user_course_enrollments = Plc::UserCourseEnrollment.all
+  end
+
+  # GET /plc/user_course_enrollments/1
+  # GET /plc/user_course_enrollments/1.json
+  def show
+  end
+
+  # GET /plc/user_course_enrollments/new
+  def new
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.new
+  end
+
+  # POST /plc/user_course_enrollments
+  # POST /plc/user_course_enrollments.json
+  def create
+    user = User.find_by_email_or_hashed_email(plc_user_course_enrollment_params[:user_email])
+
+    course = Plc::Course.find_by(id: plc_user_course_enrollment_params[:plc_course_id])
+
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.new(user: user, plc_course: course)
+
+    respond_to do |format|
+      if !user.nil? && @plc_user_course_enrollment.save
```

Comment:
Nit: `unless` (or better yet, invert the statement so the `else` case comes first)